com.github.expresslrs.ExpressLRSConfigurator.yml

```diff
@@ -0,0 +1,73 @@
+app-id: com.github.expresslrs.ExpressLRSConfigurator
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: expresslrs-configurator
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --socket=system-bus
+  - --filesystem=/tmp
+  - --filesystem=/run/udev:ro
+  - --filesystem=~/.platformio
+modules:
+  - name: git
+    make-args:
+      - INSTALL_SYMLINKS=1
+      - NO_PERL=YesPlease
+      - NO_TCLTK=YesPlease
+    make-install-args:
+      - INSTALL_SYMLINKS=1
+      - NO_PERL=YesPlease
+      - NO_TCLTK=YesPlease
+    post-install:
+      - find . -type f -name perllocal.pod -delete
+    sources:
+      - type: archive
+        sha256: 814641d7f61659cfbc17825d0462499ca1403e39ff53d76a8512050e6483e87a
+        url: https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.37.3.tar.xz
+        x-checker-data:
+          type: html
+          url: https://mirrors.edge.kernel.org/pub/software/scm/git/
+          version-pattern: git-([\d\.-]+).tar.xz
+          url-template: https://mirrors.edge.kernel.org/pub/software/scm/git/git-$version.tar.xz
+  - name: expresslrs-configurator
+    buildsystem: simple
+    build-commands:
+      - ar x expresslrs-configurator_*.deb
+      - rm expresslrs-configurator_*.deb
+      - tar xf data.tar.xz
+      - cp -r 'opt/ExpressLRS Configurator' '/app/ExpressLRS-Configurator'
+      - mkdir -p /app/share/icons/hicolor
+      - chmod -R a-s,go+rX,go-w "/app/ExpressLRS-Configurator"
+      - install expresslrs-configurator.sh /app/bin/expresslrs-configurator
+      - install -Dm644 expresslrs-configurator.png /app/share/icons/hicolor/scalable/apps/com.github.expresslrs.ExpressLRSConfigurator.png
+      - install -Dm644 com.github.expresslrs.ExpressLRSConfigurator.desktop /app/share/applications/com.github.expresslrs.ExpressLRSConfigurator.desktop
+      - install -Dm644 com.github.expresslrs.ExpressLRSConfigurator.appdata.xml /app/share/metainfo/com.github.expresslrs.ExpressLRSConfigurator.metainfo.xml
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/ExpressLRS/ExpressLRS-Configurator/releases/download/v1.5.3/expresslrs-configurator_1.5.3_amd64.deb
```

Comment:
1. an aarch64 flatpak is still built. `only-arches` doesn't do what you think it does.
2. the appdata say "cross platform" but then there is no aarch64 build? This is disappointing.
3. why not building from source instead?

Reply:
1. I thought this was for the checker only?
2. I guess it means cross platform as in mac/win/lin - I considered adding a flathub.json, but figured we could talk about this in this PR
3. Tried that, but some node deps we're causing problems back then (have been working on this for some time) https://github.com/ExpressLRS/ExpressLRS-Configurator/issues/390#issuecomment-1259269540

Reply:
> 1. I thought this was for the checker only?

Even if if was (it is not), this wouldn't change that an aarch64 flatpak would be build with x86_64 binaries.